packages/backend/src/apps/formsg/triggers/new-submission/get-mock-data.ts

```diff
@@ -54,6 +57,16 @@ async function getMockData($: IGlobalVariable) {
     const formFields = formDetails.form.form_fields as Array<FormField>
     for (let i = 0; i < formFields.length; i++) {
       if (data.responses[formFields[i]._id]) {
+        // forcefully include all checkbox options in the correct order
+        if (data.responses[formFields[i]._id].fieldType === 'checkbox') {
+          data.responses[formFields[i]._id].answerArray =
+            formFields[i].fieldOptions
+          // include the others option if available
+          if (formFields[i].othersRadioButton) {
+            data.responses[formFields[i]._id].answerArray.push('Others')
```

Comment:
FormSG includes what the user typed into "others" checkbox field as well, so we should mimc that

```suggestion
            data.responses[formFields[i]._id].answerArray.push('Others: sample input')
```

Reply:
thanks boss